core/src/main/java/org/infinispan/distribution/ch/impl/DefaultConsistentHash.java

```diff
@@ -371,6 +414,32 @@ public String prettyPrintOwnership() {
       return sb.toString();
    }
 
+   @Override
+   public void toScopedState(ScopedPersistentState state) {
+      state.setProperty(STATE_CONSISTENT_HASH, this.getClass().getName());
+      state.setProperty(STATE_NUM_OWNERS, numOwners);
+      state.setProperty(STATE_NUM_SEGMENTS, getNumSegments());;
+      state.setProperty(STATE_MEMBERS, members.size());
+      for (int i = 0; i < members.size(); i++) {
+         state.setProperty(String.format(STATE_MEMBER, i),
```

Comment:
Is the state expected to be empty? If the state already contains a CH's state, and the new CH has less members, we'll leave some extra members hanging around.